sos/report/plugins/kdump.py

```diff
@@ -8,6 +8,8 @@
 
 import os
 import platform
+import tempfile
+import subprocess
```

Comment:
Nack: do not import subprocess directly in plugin files. Use the APIs provided for this purpose by the `Plugin` class (that set the correct `Popen()` arguments and that handle all the necessary path conversions and logging).